### PR TITLE
Fixed many naming inconsistencies.

### PR DESCRIPTION
**Pull request by TylerBrinkley**
Switched string.Format's to String Interpolation.

Used var where type is obvious.

Upgraded dependency pre-release NuGet packages to latest.

Switched to using nameof operator where appropriate.

Removed unnecessary using statements.

### CODE REVIEW

**Comment by TylerBrinkley** [comment]
Sorry about this bear of a PR. In retrospect I probably should have split this up based on the types of changes.

File: Schemas.cs [hunk 1]
```diff
@@ -1,204 +1,187 @@
...
         }
-        private async Task GenerateClassFileFromSchemaAsync(string DestinationPath, string Namespace, string Section, JToken SchemaToken)
+
+        private async Task GenerateClassFileFromSchemaAsync(string destinationPath, string @namespace, string section, JToken schemaToken)
         {
             string entity;
-            StringBuilder sb = new StringBuilder();
-            string partial = (Section == "Loan") ? " partial " : "";
+            var sb = new StringBuilder();
+            var partial = (section == "Loan") ? "partial " : "";
```
Comment (hfcjweinstock) [inline Schemas.cs]: need the space here before partial

File: HelperClasses/RequestParameters.cs [hunk 2]
```diff
@@ -1,60 +1,68 @@
...
 
         public override string ToString()
         {
-            if (this.Count > 0)
+            if (Count > 0)
             {
-                StringBuilder sb = new StringBuilder();
-                sb.Append("?");
-                sb.Append(String.Join("&",this));
+                var sb = new StringBuilder();
```
Comment (hfcjweinstock) [inline HelperClasses/RequestParameters.cs]: What was wrong with string.Join?

Comment (TylerBrinkley) [inline HelperClasses/RequestParameters.cs]: I suppose internally `String.Join` uses a `StringBuilder` so I guess this change wasn't necessary though I don't think you gain anything by using a `StringBuilder` in this case. Just return the string concatenation of the ampersand with the request parameters or use an interpolated string.

Comment (TylerBrinkley) [inline HelperClasses/RequestParameters.cs]: Just changed it back to using `String.Join` and an interpolated string.

Comment (hfcjweinstock) [inline HelperClasses/RequestParameters.cs]: ok. check it in, i will double check this when I am in the office tomorrow, and merge it.

**Comment by TylerBrinkley** [comment]
Awesome, thanks!

**Comment by hfcjweinstock** [comment]
hey.  thank you for doing the work.

**Comment by TylerBrinkley** [comment]
No problem. I'd just like to get this thing up to production quality for my own uses. I'll probably be submitting more PRs going forward as I have need.

**Comment by hfcjweinstock** [comment]
great.  if it is a new functionality, let me know and i can spit up a separate branch for it.

**Comment by TylerBrinkley** [comment]
Will do.

**Comment by TylerBrinkley** [comment]
@hfcjweinstock as part of the API early access test group how did you provide feedback to the API developers. I have an issue and I'd like to ask them about it.

**Comment by hfcjweinstock** [comment]
https://hipchat.elliemae.io/ggLrhg1Wp

**Pull request merged**